onnx/defs/tensor/defs.cc

```diff
@@ -3890,4 +3890,78 @@ ONNX_OPERATOR_SET_SCHEMA(
 	        }
         }));
 
+static const char* Col2Im_ver16_doc = R"DOC(
```

Comment:
I think nn/defs.cc is a better location for this op

Reply:
done